src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -443,7 +455,7 @@ CREATE TABLE annotation
 ,created_by                  INTEGER                       NOT NULL
 ,last_modified_ts            TIMESTAMP WITH TIME ZONE      NOT NULL
 ,last_modified_by            INTEGER                       NOT NULL          
-) TABLESPACE pg_default;
+) TABLESPACE darts_tblspc;

```

Comment:
Please revert where you have used 'darts_tblspc' and replace with 'pg_default' 
This comment applies in several places in the Pull Request.